docs/content/en/_index.md

```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
 type: docs
 ---
 
-The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. Check out the sections at left based on your needs and be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
+The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. 
```

Comment:
This is probably just an idea in your head at this point... in which case let's save that convo for another day, but... What other content would you want on this intro page?  Is this where you'd want to see some intro and architecture content for operator deployed clusters?

Reply:
Yes, more common intro material will go into this page.